ConfidenceDemoApp/ConfidenceDemoApp/ConfidenceDemoApp.swift

```diff
@@ -1,4 +1,5 @@
 import ConfidenceProvider
+import Confidence
```

Comment:
This is required since we moved `InitializationStrategy` to the `Confidence` library.
Ideally we don't want the `Confidence` library to depend in any way on the `OpenFeatureProvider` library.